target/linux/ath79/dts/ar9330_teltonika_rut230.dts

```diff
@@ -0,0 +1,207 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9330.dtsi"
+
+/ {
+	compatible = "teltonika,rut230", "qca,ar9330";
+	model = "Teltonika RUT230";
+
+	chosen {
+		bootargs = "console=ttyATH0,115200 root=31:05 rootfstype=squashfs";
+	};
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &led_signal_bar2;
```

Comment:
Is this vendor behavior or did you choose the LED randomly?

Reply:
I think i was testing something, removed it.

Reply:
Okay, but now you have no LEDs for signalling boot/failsafe etc. anymore.
I'd recommend to just use the signal5 LED for those.